lib/jxl/dec_group.cc

```diff
@@ -803,12 +803,17 @@ Status DecodeGroup(BitReader* JXL_RESTRICT* JXL_RESTRICT readers,
                       PassesDecoderState::kGroupDataYBorder, dst_rect.xsize(),
                       dst_rect.ysize());
     }
+
+    // TODO(eustas): make "thread-local".
+    auto arena =
```

Comment:
Can we move this in the dec_state? There are already a few num-thread-dependent things there (IIRC PrepareForThreads is the relevant function, or something similarly named)

Reply:
Done